sdk/networkanalytics/azure-resourcemanager-networkanalytics/src/test/java/com/azure/resourcemanager/networkanalytics/NetworkAnalyticsManagerTests.java

```diff
@@ -0,0 +1,102 @@
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.Region;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.networkanalytics.models.DataProduct;
+import com.azure.resourcemanager.networkanalytics.models.DataProductProperties;
+import com.azure.resourcemanager.networkanalytics.models.ManagedServiceIdentity;
+import com.azure.resourcemanager.networkanalytics.models.ManagedServiceIdentityType;
+import com.azure.resourcemanager.resources.ResourceManager;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class NetworkAnalyticsManagerTests extends TestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.create("centraluseuap", "Central US EUAP");
+    private String resourceGroupName = "rg" + randomPadding();
+    private NetworkAnalyticsManager networkAnalyticsManager = null;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        networkAnalyticsManager = NetworkAnalyticsManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    @Disabled("Missing required property: workload. Paths in payload: '$.properties.workload'")
```

Comment:
what's this?

Reply:
The `Workload` is a required item while create data product on portal
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/0ffc8354-7ec4-4d01-8d01-cd96214ed274)
but there is no such setting interface in model [`DataProductProperties`](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/networkanalytics/azure-resourcemanager-networkanalytics/src/main/java/com/azure/resourcemanager/networkanalytics/models/DataProductProperties.java),
so disabled the live test case.

Reply:
Can you create it e.g. via CLI?

Use the `--debug` option to print out the PUT request.
If you can create it via CLI, and it provide the "workload", while we don't see it in Swagger, we need to report it to service ASAP.

Reply:
Can not create by CLI, and CLI does not support it.
https://learn.microsoft.com/en-us/cli/azure/network-analytics/data-product?view=azure-cli-latest#az-network-analytics-data-product-create

Reply:
Then create it via Portal

Reply:
> Can not create by CLI, and CLI does not support it. https://learn.microsoft.com/cli/azure/network-analytics/data-product?view=azure-cli-latest#az-network-analytics-data-product-create

Where in that page say CLI does not support it?

Reply:
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/93151541-da40-4479-9ff6-f77de7e1c854)
The parameter "workload" is not found in the parameter list of the command `az network-analytics data-product create`, and it does not use the following abnormalities will be caused:
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/fa1f2216-ad7d-4e7c-9b2b-41a86aaf00ad)

Reply:
So CLI runs and create the resource. But the resource is not correct? Please report it at https://github.com/Azure/sdk-release-request/issues/4914

How about create via Portal?

Reply:
Regardless of whether you create through the portal or the CLI, you must apply for access rights for the subscription.
https://learn.microsoft.com/en-us/azure/operator-insights/data-product-create?tabs=azure-cli#prerequisites

Reply:
Ok, then comment out the CRUD part (and no `@embedStart` `@embedEnd`).

Just leave a List, if it works. We at least need to call some API in a LIVE test.